README.md

```diff
@@ -1,48 +1,65 @@
 
-**[Dandihub](https://hub.dandiarchive.org/hub)** is a convenience JupyterHub environment that combines these resources within a virtual desktop:
-* pre-mounting the DANDI archive data and this example library\* as folders
-* pre-installing MATLAB\*\* and MatNWB software
+Users can elect **convenience MATLAB environments** which streamline all or some of these aspects, or to configure their own **individual MATLAB environment** for maximum flexibility. 
 
-Outside Dandihub, users can use this library by [cloning the whole GitHub repository](https://www.mathworks.com/help/matlab/matlab_prog/use-git-in-matlab.html) given its limited size. This can be a folder within their [MATLAB Drive](https://www.mathworks.com/products/matlab-drive.html) which makes the data available for use in [MATLAB Online](https://www.mathworks.com/products/matlab-online.html). 
-
-Utilizing [remote data access](https://www.mathworks.com/help/matlab/import_export/work-with-remote-data.html) functions for working with S3 data are recommended for the best possible performance, in both local desktop and MATLAB Online environments. 
-
-Individual examples in this library are coded and tested to access their Dandiset using one or more of these workflows (local machine, MATLAB Online, Dandihub). This will typically be described in the example README files. 
+### Convenience MATLAB Environments
+#### DandiHub MATLAB Environment
+**[Dandihub](https://hub.dandiarchive.org/hub)** is a JupyterHub environment that combines these resources within a pre-configured virtual desktop that pre-mounts the example code\* & Dandiset data as folders. MATLAB\*\* and MatNWB software are pre-installed. 
 
 (\*) coming soon <br/>
 (\*\*) using the [MATLAB Integration for Jupyter](https://www.mathworks.com/products/reference-architectures/jupyter.html)
 
-## Submitting Your DANDI Archive Example Live Scripts
+### MATLAB Online Environment
+[MATLAB Online](https://www.mathworks.com/products/matlab-online.html) is a web browser-based MATLAB environment with connectivity to [GitHub](https://www.mathworks.com/products/matlab-online/data-access.html#github) code and [AWS S3](https://www.mathworks.com/products/matlab-online/data-access.html#aws) data. 
+
+MATLAB Online uses [MATLAB Drive](https://www.mathworks.com/products/matlab-online/data-access.html#matlab-drive) as its cloud filesystem. Code can be [cloned](https://www.mathworks.com/help/matlab/matlab_prog/use-git-in-matlab.html) there from GitHub & data can be cached there from AWS.
+
+### Individual MATLAB Environment
+Individual MATLAB environments on user machines or cluster nodes offer greater storage & flexibility. Utilizing [remote data access](https://www.mathworks.com/help/matlab/import_export/work-with-remote-data.html) functions for working with S3 data is recommended for the best possible performance. 
```

Comment:
```suggestion
Individual MATLAB environments on user machines or cluster nodes offer greater storage & flexibility. Utilizing [remote data access](https://neurodatawithoutborders.github.io/matnwb/tutorials/html/remote_read.html) functions for working with S3 data is recommended for the best possible performance. 
```

Point to MatNWB tutorial on remote read

Reply:
We decided to hold off on this change for now and will discuss further, once we have more information from Mathworks and find a better place to put the general data streaming guidance